aiidalab_widgets_base/computational_resources.py

```diff
@@ -1787,6 +1792,12 @@ def __init__(
                 detailed_setup,
             ]
         )
+        # hide this conatainer if the detailed setup is enabled.
```

Comment:
I don't understand how this comment relates to the code below, perhaps it should be moved somewhere else?
```suggestion
        # hide this container if the detailed setup is enabled.
```

Reply:
Thanks for pointing out this. I updated it to
```
# This container is used to control the appearance of the password box
```